src/BinSkim.Sdk/CompilerDataLogger.cs

```diff
@@ -264,17 +274,24 @@ public static void Summarize(AnalysisSummary summary)
             }
             else
             {
-                Console.WriteLine(summary.ToString());
+                Console.WriteLine(summary);
             }
         }
 
+        internal static void Reset()
+        {
+            s_sessionId = null;
```

Comment:
>s_sessionId [](http://example.com/codeflow?start=12&length=11)

why don't you lock here?

Reply:
Added some docs in this method. We don't need a lock because it will only be called during tests.
All the tests that requires this aren't running in parallel.